feature-libs/my-account/organization/components/unit/units.config.ts

```diff
@@ -197,6 +200,25 @@ export function unitsTableConfigFactory(): TableConfig {
 
 export const unitsTableConfig: TableConfig = {
   table: {
+    [OrganizationTableType.UNIT]: {
+      cells: ['name'],
+      options: {
```

Comment:
Use the vertical layout for all screens: 

`layout: TableLayout.VERTICAL`

This will avoid the TH in each row.